config/src/network_id.rs

```diff
@@ -147,6 +152,18 @@ impl NetworkId {
         matches!(self, NetworkId::Private(network) if network == VFN_NETWORK)
     }
 
+    pub fn is_validator_network(&self) -> bool {
+        matches!(self, NetworkId::Validator)
+    }
+
+    pub fn as_network_type(&self) -> NetworkType {
+        if self.is_validator_network() {
+            NetworkType::Validator
+        } else {
+            NetworkType::FullNode
+        }
+    }
```

Comment:
oh I spoke too soon haha, looks like it's here 

I would probably rename this to just `network_type` though